File: GTP2/gtp6.cpp
#include<iostream>
#define SIZE 5

using namespace std;

float avg(int n[], int l) {
        if(l > 0) {
                return n[l-1]*1.0/SIZE + avg(n, l-1);
        } else {
                return 0;
        }
}

void prompt_vals(int ns[]) {
        for(short i = 0; i < SIZE; i++) {
                cout << "Ingrese un valor (" << SIZE - i << " restantes): ";
                cin >> ns[i];
        }
}

int main() {
        int ns[SIZE];

        prompt_vals(ns);
        
        cout << "La media aritmética es: " << avg(ns, SIZE) << "\n";

        return 0;
}

File: GTP2/gtp6.py
def main():
        numeros = []
        avg:float = 0
        # i = 5

        # while i > 0:
        #         numeros.append(int(input(f"Ingrese un valor ({i} restantes): ")))
        #         i = i-1

        # for n in numeros: avg += n/5

        numeros = [int(n) for n in input("Ingrese 5 números separados por espacios: ").split()[:5]]

        print(f"Promedio: {sum(numeros)/5}")

if __name__ == '__main__':
        main()
